src/python_testing/TC_ACL_2_6.py

```diff
@@ -136,26 +144,90 @@ async def test_TC_ACL_2_6(self):
         asserts.assert_equal(result[0].Status, Status.Success, "Write should have succeeded")
 
         self.step(5)
-        # Create correct event path with endpoint 0
-        events_response2 = await self.default_controller.ReadEvent(
-            nodeid=self.dut_node_id,
-            events=event_path,
-            fabricFiltered=True,
-            eventNumberFilter=max(initial_event_num)+1
-        )
+        # Wait for subscription events
+        received_subscription_events = []
+        for _ in range(2):  # Expect 2 events
+            event_data = events_callback.wait_for_event_report(acec_event, timeout_sec=15)
+            logging.info(f"Received subscription event: {event_data}")
+            received_subscription_events.append(event_data)
 
-        logging.info(f"Events response: {events_response2}")
-        logging.info(f"Event response length: {len(events_response2)}")
-        asserts.assert_true(len(events_response2) == 2, "Expected 2 events")
+        # Read events with retries
```

Comment:
Since you've already got the event via a subscription, you won't need the retries here any more.